### ISSUE TITLE
Bad output for arrays with line breaks in rows

### ISSUE THREAD

**Issue by m-kessler** [ErykMroczek/mofmt#9]
mofmt usually formats arrays nicely.
However, it has problems with line breaks in row definitions, which is especially a problem for Dymola users.

Here we see what Dymola generates when you define a matrix of size 2x16 via the GUI (with the default maximum line length of 80):
```
model array
  parameter Real a[:,:]=[1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1; 2,2,2,2,2,2,2,2,2,2,2,
      2,2,2,2,2];
end array;
```

Running the code through mofmt gives:
```
model array

  parameter Real a[:,:] = [
    1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1;

      2,
      2,
      2,
      2,
      2,
      2,
      2,
      2,
      2,
      2,
      2,
      2,
      2,
      2,
      2,
      2];

end array;
```

**Comment by ErykMroczek** [comment]
Well, looks like it doesn't work like charm in the end.

Matrices are something I have planned to work on. That empty line and wrong indentation are definitely bugs and need fixing, but I am not sure how to handle such long rows. In this case the result is kinda ugly, yet current approach (placing every member in a separate line) is consistent with the way that other syntactically similar constructs are handled (function arguments etc.) Moreover, for example `Black` similarly handles long lists in Python.

So, I will definitely fix the issue with an empty line and bad indentation, but as for the formatting style as such - I am not sure yet. In this case user needs to manually remove the break in the second row, and insert another break between rows to get a better result.

**Comment by m-kessler** [comment]
Yes, this is a bit tricky. I like that mofmt tries to preserve existing wrapping and that it does not use a maximum line length.
So doing a manual edit in this situation sounds ok to me. The main reason to open the issue were the formatting issues you described (empty line, bad indentation).
